neard/src/cli.rs

```diff
@@ -283,6 +286,26 @@ pub(super) struct RunCmd {
     max_gas_burnt_view: Option<Gas>,
 }
 
+/// SystemStoppedActor is used to monitor whether the actix System was stopped.
+struct SystemStoppedActor {}
+
+impl Actor for SystemStoppedActor {
+    type Context = Context<Self>;
+
+    // Does nothing but needs to be implemented for the Actor trait trait.
+    fn started(&mut self, _: &mut Self::Context) {}
+}
+
+use once_cell::sync::Lazy;
+pub static SYSTEM_STOPPED_CHANNEL: Lazy<Channel<bool, [bool; 1]>> = Lazy::new(|| Channel::new());
```

Comment:
I gave it a try and couldn't make it work.
Stuck at this panic:

```
thread 'main' panicked at 'Cannot start a runtime from within a runtime. This happens because a function (like `block_on`) attempted to block the current thread while the thread is being used to drive asynchronous tasks.',
```